pkgs/development/tools/xcpretty/update

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p bash ruby bundler bundix
+
+rm Gemfile.lock
+bundler install
```

Comment:
If you do `bundle lock --update` here, you don't need to remove the Gemfile, and Bundler won't actually attempt to install the gems. This means that native dependencies won't cause problems, and you won't need the `~/.gem` hack, since all gem builds will happen under Nix through Bundix.

Reply:
😮 cool